packages/itwinui-css/src/table/base.scss

```diff
@@ -371,8 +371,7 @@
   > .iui-table-empty {
     text-align: center;
     position: sticky;
-    inset-inline-start: 50%;
-    transform: translateX(-50%);
+    margin-inline: auto;
```

Comment:
Although this fixes the problem mentioned in the attached issue, I guess there are some edge cases where the empty content is not centered.

Maybe those cases cannot be addressed because the table is narrower than the empty content width and thus, we do not need to cover this case?

<details>
<summary>Example un-centered case when the table is very narrow</summary>

https://github.com/iTwin/iTwinUI/assets/45748283/8e71a970-3a79-4bb8-9186-45c6fc319515

</details>

Reply:
I tried dropping one of our illustration SVGs inside the empty state and it seemed to work ok.
![table-empty-state-resize](https://github.com/iTwin/iTwinUI/assets/849817/2d073fd3-1380-43ff-b260-e1fb913b90df)

Perhaps their SVG has a hardcoded size?  Just confirmed adding `width='400px'` to the `<svg>` does indeed break it.